lib/error_tracker.ex

```diff
@@ -180,4 +171,18 @@ defmodule ErrorTracker do
   def get_context do
     Process.get(:error_tracker_context, %{})
   end
+
+  defp normalize_exception(%struct{} = ex, _stacktrace) when is_exception(ex) do
+    {to_string(struct), Exception.message(ex)}
+  end
+
+  defp normalize_exception({kind, ex}, stacktrace) do
```

Comment:
An exception can also be an atom, such as an `ArithmeticError`. This happened to me just now. 

Exception:

    ** (CaseClauseError) no case clause matching: :badarith
        (error_tracker 0.1.2) lib/error_tracker.ex:105: ErrorTracker.report/3
        (error_tracker 0.1.2) lib/error_tracker/integrations/plug.ex:102: ErrorTracker.Integrations.Plug.report_error/3
        (stocker 0.1.0) lib/stocker_web/router.ex:1: StockerWeb.Router.call/2
        (stocker 0.1.0) lib/stocker_web/endpoint.ex:1: StockerWeb.Endpoint.plug_builder_call/2
        (stocker 0.1.0) deps/plug/lib/plug/debugger.ex:136: StockerWeb.Endpoint."call (overridable 3)"/2
        (stocker 0.1.0) lib/stocker_web/endpoint.ex:1: StockerWeb.Endpoint."call (overridable 4)"/2
        (stocker 0.1.0) lib/stocker_web/endpoint.ex:1: StockerWeb.Endpoint.call/2
        (phoenix 1.7.11) lib/phoenix/endpoint/sync_code_reload_plug.ex:22: Phoenix.Endpoint.SyncCodeReloadPlug.do_call/4
        (bandit 1.5.5) lib/bandit/pipeline.ex:124: Bandit.Pipeline.call_plug!/2
        (bandit 1.5.5) lib/bandit/pipeline.ex:36: Bandit.Pipeline.run/4
        (bandit 1.5.5) lib/bandit/http1/handler.ex:12: Bandit.HTTP1.Handler.handle_data/3
        (bandit 1.5.5) lib/bandit/delegating_handler.ex:18: Bandit.DelegatingHandler.handle_data/3
        (bandit 1.5.5) /data/home/jim/git/stocker/deps/thousand_island/lib/thousand_island/handler.ex:379: Bandit.DelegatingHandler.handle_info/2
        (stdlib 4.3.1.4) gen_server.erl:1123: :gen_server.try_dispatch/4
        (stdlib 4.3.1.4) gen_server.erl:1200: :gen_server.handle_msg/6
        (stdlib 4.3.1.4) proc_lib.erl:240: :proc_lib.init_p_do_apply/3

Reply:
Thanks for the review! I was able to reproduce it and it was a bug introduced during a refactor made when finishing this change.

I can confirm it already works on my side with the latest commit (https://github.com/elixir-error-tracker/error-tracker/pull/36/commits/28c6c2b453ee2325521d68bfa79bd952c538d966). Could you check if your issue is fixed too? 

Thanks

Reply:
Yes, I will check it now. Thanks for the quick response. Can't wait to use this in production!

Reply:
Works great!